lib/impress.application.js

```diff
@@ -187,7 +189,7 @@ impress.application.mixin = (application) => {
   }
 
   application.createSandbox = (callback) => {
-    const sandbox = { module: {}, api: {} };
```

Comment:
I guess the deprecation of `__filename`, `__dirname` and `module` should be documented in, at least, commit message.